src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -58,7 +58,7 @@
     <None Update="libraries\System.IO.Compression\TestData\**">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="libraries\System.Utf8String.Experimental\*.txt">
+    <None Update="libraries\System.Utf8String\*.txt">
```

Comment:
@adamsitnik Shouldn't the folder names match the assembly name?

Reply:
@carlossanlop This is a very good question, however, I am not sure if there is a good answer to it.

The namespace, class name, method name, argument name and value are used to build benchmark ID which is used in the reporting system. Something like `namespace.className.methoName(argName: argValue)`. Example: https://pvscmdupload.blob.core.windows.net/reports/allTestHistory/refs/heads/master_x64_Windows%2010.0.18362/Microsoft.Extensions.Primitives.StringSegmentBenchmark.EndsWith.html

So first of all, we need to choose good names because they are used as IDs to identify benchmarks: https://github.com/dotnet/performance/blob/master/docs/microbenchmark-design-guidelines.md#Benchmarks-are-Immutable

Here I hope that in 6.0 we are simply going to switch from `Experimental` to just not experimental and I would prefer to avoid using `Experimental` in the name.

The other problem is that this particular namespace is used to copy the output text files to output folder and the longer it is, the higher chance of hitting `too long name` path exceptions etc. So in general, the shorter names, the better.